src/app/search/loadouts/search-filters/freeform.ts

```diff
@@ -33,8 +33,10 @@ const freeformFilters: FilterDefinition<
     keywords: ['name', 'exactname'],
     description: tl('LoadoutFilter.Name'),
     format: 'freeform',
-    suggestionsGenerator: ({ loadouts }) =>
-      loadouts?.map((loadout) => `exactname:${quoteFilterString(loadout.name.toLowerCase())}`),
+    suggestionsGenerator: ({ loadouts, selectedLoadoutsStore }) =>
+      loadouts
+        ?.filter((loadout) => loadout.classType === selectedLoadoutsStore?.classType)
```

Comment:
I wonder if we should just hoist this filtering of loadouts to the context scope? I can't see why we wouldn't want to, so unless there objections I will do it in a follow up PR

Reply:
Yeah I think that would be best.